README.md

```diff
@@ -2,4 +2,22 @@
 
 Continuous Delivery
 
-README coming soon!
+## How to deploy to stage
+
+- bump nox or cli version in `versions.json` or make changes in `deployment/*`
+- create and merge PR with changes
+
+Or merge renovate PR.
+
+## How to deploy to testnet/kras
+
+- merge release PR
+- approve deployment in `Actions` tab
+
+## How to deploy tag or branch
+
+Go to `Actions` -> `deploy` and press `Run workflow`
+
+## How to cleanup nox volumes
+
+Add a label `cleanup` to a PR before merge.
```

Comment:
will it clean testnet/kras as well? or only stage?

I think only stage should be enough

Reply:
sounds a bit dangerous to me, but I don't have a better solution

Reply:
Yes, this is not an ideal solution, but a quick one! I'm tired of recreating volumes by hand :)